app/src/main/java/com/zulip/android/models/Person.java

```diff
@@ -30,8 +28,8 @@
     public static final String ISACTIVE_FIELD = "isActive";
     private static final String ID_FIELD = "id";
     private static final String AVATARURL_FIELD = "avatarUrl";
-    @SerializedName("IGNORE_MASK")
-    @DatabaseField(columnName = ID_FIELD, generatedId = true)
+    @SerializedName("user_id")
+    @DatabaseField(columnName = ID_FIELD, id = true)
```

Comment:
Why not have the ID_FIELD and SerializedName ?
To avoid confusion, in received json and elsewhere?